db/migrate/20230822114552_add_links.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class AddLinks < ActiveRecord::Migration[6.1]
+  def change
+    create_table :links do |t|
+      t.string :title
```

Comment:
Én csak az issue-ban lévő form alapján gondolom így, de ha a `title` és az `url` nem lehet üres, akkor ezt garantáljuk az adatbázis szintjén is légyszi!